### ISSUE TITLE
Material 1.0.x support

### ISSUE THREAD

**Issue by yanickrochon** [Zetoff/accounts-material-ui#36]
At some point, Material v1.0.x will have to be supported because the project is already at the pre-release stage at this time.

I don't see a lot of breaking changes, apart from the obvious color API changes.

**Comment by yanickrochon** [comment]
I have attempted rewriting the component, and it's a little more than fixing imports. I would also replace FontAwesome with actual React components, notably icons from [mdi-material-ui](https://github.com/TeamWertarbyte/mdi-material-ui)

**Comment by yanickrochon** [comment]
First commit, [here](https://github.com/yanickrochon/accounts-material-ui/tree/material-ui-v1) for testing. I did remove unnecessary dependencies (probable fix for #34?) Did not implement the PR #33.

Comments welcome, and hopefully a PR whenever possible.

**Comment by PolGuixe** [comment]
I agree we need to move to Material-UI 1.0.x. I'll try to get some work done on this in the coming weeks as I need it for a project.

I'll have a look at PR #33.

**Comment by yanickrochon** [comment]
Great! I just started a project and need it as well. As I upgraded my branch to 1.x of MUI, there is now a Grid component that should be part of the default layout, and I wil make another commit, tonight about this. Comments welcome! If all is alright with you, this branch can then be merged into a `next` branch in this repository and continue to work from there. I'm more than happy to contribute.

**Comment by yanickrochon** [comment]
What's the status of this?
